Formula/libsndfile.rb

```diff
@@ -1,6 +1,24 @@
 class Libsndfile < Formula
   desc "C library for files containing sampled sound"
   homepage "http://www.mega-nerd.com/libsndfile/"
+  # Change back to stable when upstream releases > 1.0.28
+  # Fixes several security vulnerabilities:
+  #  CVE-2017-6892
+  #  CVE-2017-8361
+  #  CVE-2017-8362
+  #  CVE-2017-8363
+  #  CVE-2017-8365
+  #  CVE-2017-12562
+  #  CVE-2017-14245
+  #  CVE-2017-14246
+  #  CVE-2017-14634
+  #  CVE-2018-13139
+  #  CVE-2018-13419
+  #  CVE-2018-19432
+  #  CVE-2018-19661
+  #  CVE-2018-19662
+  #  CVE-2018-19758
+  #  CVE-2019-3832
```

Comment:
```suggestion
  # Fixes several security vulnerabilities, for example:
  # CVE-2017-6892, CVE-2017-12562, CVE-2018-13139
```

I don’t see how enumerating every single related CVE helps, other than to prove a point – which is not necessary as we’re already considering your submission, and given that upstream has the fixes merged to `master` already.

CVE-2017-6892, CVE-2017-12562 and CVE-2018-13139 are [the three most critical ones](https://www.cvedetails.com/vulnerability-list.php?vendor_id=16294&product_id=36889&version_id=213074&page=1&cvssscoremin=6).